lib/diagnostic.rb

```diff
@@ -13,28 +19,35 @@ def longest(enum)
   # enumerable supplied (i.e. apply the `+` operator)
   # Assume each entry implements a `+` method
   def sum(enum)
-    # Your code goes here
+    enum.reduce(0) { |m, p| m += p }
   end
 
   # Question 3
   # Write ruby code that returns true if some of the entries
   # in the numbers enumerable are odd
   def some_odd(numbers)
-    # Your code goes here
+    numbers.each do |num|
+      if num.odd?
+        return true
+      end
+    end
```

Comment:
This will do the desired behavior. But as in Q1, there's an easier way : `.any?`
e.g.
`numbers.any? { |num| num.odd? }` or `numbers.any?(&:odd?)`